extensions/sql-migration/src/constants/strings.ts

```diff
@@ -356,6 +356,9 @@ export const TARGET_PLATFORM = localize('sql.migration.target.platform', "Target
 export const WARNINGS_DETAILS = localize('sql.migration.warnings.details', "Warnings Details");
 export const ISSUES_DETAILS = localize('sql.migration.issues.details', "Issue Details");
 export const SELECT_DB_PROMPT = localize('sql.migration.select.prompt', "Click on SQL Server Instance or any of the databases on the left to view its details.");
+export const BLOCKING_ISSUE_LOG_FILES = localize('sql.migration.blocking.issue.log', "MultipleLogFiles");
```

Comment:
I don't think we need to localize these ids. Only the strings that are displayed need to be localized. Just make an array of blocking IDs and check if the id is in that array.